graviton/blackbox.py

```diff
@@ -49,7 +48,9 @@ class ExecutionMode(Enum):
 
 
 class DryRunProperty(Enum):
-    cost = auto()
+    cost = auto()  # deprecated
```

Comment:
I'm not particularly concerned about backwards compatibility in this case. It would only be a problem if users were relying on bad cost information to begin with. ie I wouldn't mind keeping cost around as a derived field.